resolver/graph.go

```diff
@@ -211,33 +211,32 @@ func (g *MessageDependencyGraph) createMessageResolver(ctx *context, node *Messa
 	}
 	ctx.addError(
 		ErrWithLocation(
-			fmt.Sprintf(`%q message has not resolver content`, node.Message.Name),
-			source.MessageLocation(ctx.fileName(), node.Message.Name),
```

Comment:
The validation nodes do not have a Message reference, so I needed to change it from `node.Message.Name` to `node.BaseMessage.Name`. Considering the context, I believe node.BaseMessage.Name is more appropriate than node.Message.Name since that is where the resolver rule should have been defined but WDYT? 🤔

Reply:
The same with some of the similar changes below

Reply:
Thank you ! `BaseMessage` is correct !